packages/flutter/test/material/tabs_test.dart

```diff
@@ -14,6 +14,7 @@ import '../rendering/mock_canvas.dart';
 import '../rendering/recording_canvas.dart';
```

Comment:
Before and after: 
<img width="500" src="https://user-images.githubusercontent.com/7915388/103940821-d9b0b680-50e2-11eb-9773-522c93849203.png">
<img width="653" alt="Screen Shot 2021-01-07 at 12 24 16 PM" src="https://user-images.githubusercontent.com/7915388/103941157-4fb51d80-50e3-11eb-9ff1-c5493c341a6b.png">